api/tests/test_user.py

```diff
@@ -0,0 +1,131 @@
+from django.test import TestCase
+
+from django.contrib.auth import get_user_model
+
+from ..forms import UserCreationForm, UserChangeForm
+
+User = get_user_model()
+
+
+class UserTestCase(TestCase):
+    def setUp(self):
+        self.user = User.objects.create(
+            email='test@site.com', name='test_user',
+            cohort=20, slack_handle='@test_user')
+
+    def test_can_add_user(self):
+        users_count_before = User.objects.all().count()
```

Comment:
Any reason why you did not use `User.objects.count()` . It seem shorter and serves the same purpose. Same to line 22